StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -24,12 +25,13 @@ private let paymentMethodLogoSize: CGSize = CGSize(width: 54, height: 40)
 /// For internal SDK use only
 @objc(STP_Internal_SavedPaymentMethodCollectionView)
 class SavedPaymentMethodCollectionView: UICollectionView {
-    init(appearance: PaymentSheet.Appearance) {
+    init(appearance: PaymentSheet.Appearance, showDefaultPMBadge: Bool = false) {
```

Comment:
Variable name is awkward since we technically aren't showing a default PM badge in this context -- I think we should rename this to something like, `needsVerticalPaddingForBadge`.  If we end up needing a 3rd size, then we should change it from a Bool to a Float and call it `verticalPaddingForBadge` and just calculate it.